server/db/Order.js

```diff
@@ -28,16 +29,39 @@ Order.getActiveOrderByUser = (userId) => {
   })
 }
 
-Order.addLineItem = ({userId, product}) => {
-  User.findById(userId, { include: Order })
+Order.createLineItem = ({orderId, productId}) => {
+  LineItem.findOne({
+    where: {
+      productId: productId
```

Comment:
There can be multiple lineitems with that productId. You need to find one that has the productId and OrderId